cvmfs_ducc: unable to redefine 'n' shorthand in "garbage-collection" flagset: it's already used for "dry-run" flag

```sh
$ cvmfs_ducc gc help
panic: unable to redefine 'n' shorthand in "garbage-collection" flagset: it's already used for "dry-run" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc0000b5900, 0xc00038c5a0)
	/root/go/pkg/mod/github.com/spf13/pflag@v1.0.3/flag.go:862 +0x3f5
github.com/spf13/pflag.(*FlagSet).AddFlagSet.func1(0xc00038c5a0)
	/root/go/pkg/mod/github.com/spf13/pflag@v1.0.3/flag.go:875 +0x65
github.com/spf13/pflag.(*FlagSet).VisitAll(0x12, 0xc00025fc80)
	/root/go/pkg/mod/github.com/spf13/pflag@v1.0.3/flag.go:278 +0xe3
github.com/spf13/pflag.(*FlagSet).AddFlagSet(0xbc3580, 0x2)
	/root/go/pkg/mod/github.com/spf13/pflag@v1.0.3/flag.go:873 +0x3d
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xbc3580)
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:1495 +0x65
github.com/spf13/cobra.stripFlags({0xc0002f3f20, 0x1, 0x1}, 0x1)
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:474 +0x5b
github.com/spf13/cobra.(*Command).Find.func1(0xc000020190, {0xc0002f3f20, 0x1, 0x1})
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:533 +0x49
github.com/spf13/cobra.(*Command).Find.func1(0xbc3a40, {0xc000020190, 0x2, 0x2})
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:541 +0xb1
github.com/spf13/cobra.(*Command).Find(0x7fff3ef49de0, {0xc000020190, 0xc00005e708, 0x4400f1})
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:546 +0x53
github.com/spf13/cobra.(*Command).ExecuteC(0xbc3a40)
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:833 +0x150
github.com/spf13/cobra.(*Command).Execute(...)
	/root/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/cvmfs/ducc/cmd.EntryPoint(...)
	/root/cvmfs/ducc/cmd/root.go:37
main.main()
	/root/cvmfs/ducc/main.go:8 +0x25
```

Cause:

https://github.com/cvmfs/cvmfs/blob/e54432cbd2deb065947b6f7f1f28ca0765019461/ducc/cmd/root.go#L19

Conflict with:

https://github.com/cvmfs/cvmfs/blob/e54432cbd2deb065947b6f7f1f28ca0765019461/ducc/cmd/garbage_collection.go#L27

Because `--notification-file` is a persistent flag, the shorthand `-n` is already used and cannot be reused for `--dry-run`.

A simple fix:

```diff
-	garbageCollectionCmd.Flags().BoolVarP(&dryRun, "dry-run", "n", false, "Dry run the garbage collection")
+	garbageCollectionCmd.Flags().BoolVar(&dryRun, "dry-run", false, "Dry run the garbage collection")
	rootCmd.AddCommand(garbageCollectionCmd)
```

--- a/ducc/cmd/garbage_collection.go
+++ b/ducc/cmd/garbage_collection.go
@@ -24,7 +24,7 @@ var (
 )
 
 func init() {
-	garbageCollectionCmd.Flags().BoolVarP(&dryRun, "dry-run", "n", false, "Dry run the garbage collection")
+	garbageCollectionCmd.Flags().BoolVar(&dryRun, "dry-run", false, "Dry run the garbage collection")
 	rootCmd.AddCommand(garbageCollectionCmd)
 }
 

